clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -733,7 +733,7 @@ def ext_main_returns_nonint : ExtWarn<"return type of 'main' is not 'int'">,
     InGroup<MainReturnType>;
 def note_main_change_return_type : Note<"change return type to 'int'">;
 def err_main_surplus_args : Error<"too many parameters (%0) for 'main': "
-    "must be 0, 2, or 3">;
+    "must be 0">;
```

Comment:
This is a generic change, right? This may be an issue if we want our clang to be used to compile host code.

Reply:
You're right, need something more dpu specific